tools/qontract_cli.py

```diff
@@ -4320,6 +4321,43 @@ def migrate(ctx, dry_run: bool, skip_build: bool) -> None:
     rich_print(f"[b red]Please remove the temporary directory ({tempdir}) manually!")
 
 
+@external_resources.command()
+@binary(["docker"])
+@click.pass_context
+def debug_shell(ctx) -> None:
+    """Enter an ERv2 debug shell to manually migrate resources."""
+    # use a temporary directory in $HOME. The MacOS colima default configuration allows docker mounts from $HOME.
+    tempdir = Path.home() / ".erv2-debug"
+    tempdir.mkdir(exist_ok=False)
```

Comment:
if any exception throw in `enter_shell`, this temp dir won't be auto deleted, next run will fail here, should we use `tempfile.TemporaryDirectory` and `with` to let it auto delete?

Reply:
Yes, for sure 🤦 Changed in 009dbe1